src/main/java/org/benetech/servicenet/service/impl/OrganizationMatchServiceImpl.java

```diff
@@ -150,11 +148,19 @@ public void createOrUpdateOrganizationMatches(Organization organization) {
 
     private List<OrganizationMatch> findAndPersistMatches(Organization organization, List<Organization> notMatchedOrgs) {
         List<OrganizationMatch> matches = new LinkedList<>();
+        long startTime = System.currentTimeMillis();
+        log.debug("Searching for matches for " + organization.getAccount().getName() + "'s organization '" +
+            organization.getName() + "' has started. There are " + notMatchedOrgs.size() + " organizations to compare with");
         for (Organization partner : notMatchedOrgs) {
             if (isSimilar(organization, partner)) {
                 matches.addAll(createOrganizationMatches(organization, partner));
             }
         }
+        long stopTime = System.currentTimeMillis();
```

Comment:
please add TODO for removing this logic in the future, you cloud also add a link to an issue for refactoring the logs.